app/presenters/portfolio_case_presenter.rb

```diff
@@ -1,6 +1,17 @@
 class PortfolioCasePresenter < SimpleDelegator
-  def created_at
-    I18n.localize(tribunal_case.created_at, format: :portfolio)
+  def expires_in
+    I18n.translate!(:case_expires_in, scope: 'time', count: case_remaining_days)
+  end
+
+  def expires_in_class
+    case case_remaining_days
+    when 0
```

Comment:
should this be "when <= 0" just in case something goes wrong with the delete jobs?